src/panels/lovelace/cards/energy/hui-energy-distribution-card.ts

```diff
@@ -234,6 +261,19 @@ class HuiEnergyDistrubutionCard
                       kWh
                     </span>`
                   : ""}
+                <span class="consumption">
+                  ${hasReturnToGrid
+                    ? html`<ha-svg-icon
+                        class="small"
+                        .path=${mdiArrowRight}
```

Comment:
![image](https://user-images.githubusercontent.com/1444314/128807516-fe00f923-27f9-45e6-b0b8-312fe4db67b1.png)

I find the arrows a bit confusing here. In the grid we're having the arrows aligned with the lines.

Reply:
Also the kWh units are not aligned

Reply:
You want the arrows up and down here?

Reply:
yes